pallets/streaming/src/lib.rs

```diff
@@ -100,17 +100,19 @@ pub mod pallet {
         /// Insufficient deposit size
         DepositLowerThanMinimum,
         /// Start time is before current block time
-        StartBeforeBlockTime,
-        /// Stop time is before start time
-        StopBeforeStart,
+        StartTimeBeforeCurrentTime,
+        /// End time is before start time
+        EndTimeBeforeStartTime,
         /// The duration calculated is too short or too long
         InvalidDuration,
+        /// The rate per second calculated is zero
+        InvalidRatePerSecond,
         /// The stream id is not found
         InvalidStreamId,
-        /// Caller is not the streamer
-        NotTheStreamer,
-        /// Caller is not the recipient
-        NotTheRecipient,
+        /// Caller is not the stream sender
+        NotTheStreamSender,
```

Comment:
can be simplified as:

- NotTheSender
- NotTheRecipient

Reply:
yep